packages/manager/modules/sign-up/src/form/details/details.controller.js

```diff
@@ -115,6 +115,35 @@ export default class SignUpDetailsCtrl {
     return zipCode;
   }
 
+  static setInputValidity(inputFormName, form) {
+    const inputField = form[inputFormName];
+
+    if (!inputField) {
+      throw new Error(
+        `Sign up form account: Input ${inputFormName} not found in the form`,
+      );
+    }
+
+    const inputModelValue = inputField.$modelValue;
+    const inputViewValue = inputField.$viewValue;
+
+    const inputModelValid = inputField.$validators.pattern(
+      inputModelValue,
+      inputViewValue,
+    );
+
+    inputField.$setValidity('pattern', inputModelValid);
+  }
+
+  preselectLanguage() {
+    if (get(this.signUpFormCtrl, 'rules.language.in.length', 0) === 1) {
+      const [uniqueLanguage] = this.signUpFormCtrl.rules.language.in;
```

Comment:
Do you know why it's not autoselected by this part of code?
https://github.com/ovh/manager/blob/8ad05e5e3a1a68e579ad64ae1f02198d8b78cc4f/packages/manager/modules/sign-up/src/form/form.controller.js#L95-L100

Reply:
It seems that the language option list contains a blank item. Don't know if it's related to this, but I will check again